.github/workflows/CI.yml

```diff
@@ -34,11 +34,15 @@ jobs:
           - macos-latest
           - windows-latest
         arch:
-          - x64
+          - default
         include:
           - os: ubuntu-latest
             version: '1'
             arch: x86
```

Comment:
The `arch` key is now unused, so this job is actually duplicating the other `ubuntu-latest` with version `1`.

Reply:
That doesn't seem to match what the CI logs here report, one Julia identifies as 32bit the other as 64bit, as expected.